packages/geo/src/lib/layer/shared/layers/layer.interface.ts

```diff
@@ -40,6 +40,8 @@ export interface GeoWorkspaceOptions {
   minResolution?: number;
   maxResolution?: number;
   enabled?: boolean;
+  noQueryOnClickInTab?: boolean;
+  noMapQueryOnOpenTab?: boolean;
```

Comment:
Shouldn't it be defined in the workspace object ? That way, you could only check that condition on workspace query (igo2 assemblage).

Reply:
Yes, but the workspaceOptions which is used at the creation of the object are in workspaceOptions (common/.../workspace.interface.ts). I think these variables need to be defined in workspace options and not in the layer workspace options.